lib/Common/Memory/Recycler.cpp

```diff
@@ -1298,6 +1314,28 @@ bool Recycler::AllowNativeCodeBumpAllocation()
 }
 
 
+#if DBG
+void Recycler::VerifyPageHeapFillAfterAlloc(char* memBlock)
```

Comment:
done

---
In reply to: [60989848](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60989848) [](ancestors = 60989848)